kamon-core/src/main/scala/kamon/trace/logging/MdcKeysSupport.scala

```diff
@@ -34,7 +39,12 @@ trait MdcKeysSupport {
 
   private[this] def copyToMdc(traceContext: TraceContext): Iterable[String] = traceContext match {
     case ctx: MetricsOnlyContext ⇒
-      ctx.traceLocalStorage.underlyingStorage.collect {
+
+      // Add the default key value pairs for the trace token and trace name.
```

Comment:
@dspasojevic @ivantopo guys I think that instead of using:
```scala

val traceTokenKey = "traceToken"
val traceNameKey = "traceName"

private val defaultKeys = Seq(traceTokenKey, traceNameKey)

MDC.put(traceTokenKey, ctx.token)
MDC.put(traceNameKey, ctx.name)
 ```
we can use the kamon api like this:

```scala
val TraceTokenKey = AvailableToMdc.fromKey("traceToken")
val TraceNameKey = AvailableToMdc.fromKey("traceName")

TraceLocal.store(TraceTokenKey)(ctx.token)
TraceLocal.store(TraceNameKey)(ctx.name)
```
this is only a idea, please guys WDYT?